src/liboslnoise/wide/wide_gabornoise.h

```diff
@@ -41,7 +41,8 @@ wide_gabor(Masked<Dual2<float>> wResult, Wide<const Dual2<float>> wX,
         sfm::GaborUniformParams gup(*opt);
         __OSL_SETUP_WIDE_DIRECTION
 
-#if !OSL_NON_INTEL_CLANG  // Control flow too complex for clang's loop vectorizor
+#if (!OSL_ANY_CLANG || OSL_INTEL_COMPILER)
```

Comment:
Aside, that we don't need to address in this PR:

There are a lot of instances of this construct that means "if we trust omp simd for complex loops". If a future clang is better with these, we will need to not only change this in quite a few locations, but also it'll need to be a more complicated test that checks which specific version of which clang compilers.

Perhaps it is better to JUST ONCE, in platform.h, have a single

```
#if OSL_OPENMP_SIMD && (OSL_GCC_VERSION || OSL_INTEL_COMPILER || OSL_INTEL_CLANG_VERSION || ...etc...)
    #define OSL_OMP_SIMD_COMPLEX_LOOP OSL_PRAGMA(omp simd simdlen(__OSL_WIDTH))
#else
    #define OSL_OMP_SIMD_COMPLEX_LOOP
#endif
```

and then all these spots can simply say

```
    OSL_OMP_SIMD_COMPLEX_LOOP
    for (...)
```

So in a future where certain particular versions of clang are known to handle these loops well, then we can modify the single master definition and have all the loops do the right thing with no change of cut-and-paste errors or missing any.

Reply:
The OSL_NON_INTEL_CLANG always bugged me when reading it, and it got worse with the new mix of classic and llvm based compilers.  So I would just go ahead and make this change now and feel better about it!

Reply:
I still like to pass arguments vs. assuming simdlen(__OSL_WIDTH), but perhaps something like 
```
#define OSL_OMP_SIMD_LOOP(...) OSL_OMP_PRAGMA(omp simd __VA_ARGS__)
#if (OSL_GCC_VERSION || OSL_INTEL_COMPILER)
    #define OSL_OMP_COMPLEX_SIMD_LOOP(...) OSL_OMP_PRAGMA(omp simd __VA_ARGS__)
#else
     // Ignore requests to vectorize complex SIMD loops for certain compiler's/versions
    #define OSL_OMP_COMPLEX_SIMD_LOOP(...)
#endif
```

Reply:
I think that's reasonable, yes.